Add: New map "Celestation"

SAADf603 opened the pull request:
##Описание
Первая Мульти-З карта для основной ротации SS1984.
Оригинальная карта: https://github.com/ParadiseSS13/Paradise/pull/19951
Автор оригинальной карты: [S34NW](https://github.com/S34NW)

Добавляет станцию Селестия.
Карта сохраняет оригинальный "дух" Фарагуса - разбросанность, изолированность, ощущение, что станция делалась в астероидах, что бы сэкономить на постройке.
От предшественников карта отличается:

1. Больше никаких коридоров длиной в сектор. 
Все основные переходные между астероидами коридоры заменены на трансферные трубы, которые быстро и безопасно (по заверениям конструкторов станции) доставят экипаж на СОСЕДНИЕ астероиды. Так же была расширена сеть квантум падов: каждый астероид теперь имеет свой квантум пад отдел, который доставит на любой НЕ СОСЕДНИЙ астероид. Исключением является Инженерный астероид, пады которого соединены с КАЖДЫМ астероидом.
В дополнение к трансферным трубам были проложены технические коридоры между астероидами, которые помогут в передвижении, если трансферные трубы были уничтожены.

 2. Конечно же мульти-З. Карта имеет три стационарных уровня:
Первый - Основания астероидов. По большей части это заброшенные технические помещения, которые остались от раскопок при изучении астероидов, однако некоторые отделы станции так или иначе расположены в основании.  
Второй - Основной уровень станции. Большую часть карты занимает именно этот уровень. 
Третий - Верхний уровень. Используется для того, что бы расширить пространство отделов, не создавая огромных коридоров, и создать новые маршруты и новые помещения.

 3. Пересмотрена система питания станции:
Инженерный астероид теперь разделён на две части - Основная инженерия и Зона двигателя. Взрыв кристалла супер материи теперь не будет нести столь критический урон для станции. Солнечные панели помогут инженерам 
поддерживать питание станции.

 4. Пересмотрена система мусоропровода:
 Мусоропровод всё так же имеет внешний контур, однако он будет в единой замкнутой системе. Пролетая полный круг из карго персонаж получает порядка 85 урона, что будет недостаточно для гибели персонажа, но достаточно, что бы побег был затруднён. Вам придётся дважды подумать, действительно ли вы хотите прокатиться по мусорной трубе.
 
 5. Спутник ИИ - отдельный шаттл, который был доставлен в сектор отдельно, как и на прочих картах, получил ряд улучшений. ИИ получил больше возможностей для управления спутником.
 
 6. Робототехники получили собственный телепортер.
 Это сделано для удобства доставки мехов в различные астероиды. Единственным ограничением является то, что самостоятельно робототехники не смогут его настроить. Для этого потребуется персонаж с уровнем доступа "Директор Исследований"


TO DO:

- [x] muti-z buttons work
- [x] autoname cameras fix
- [x] fliped railings corners (no need)
- [x] door_timer fix

SAADf603 committed:
close as never before

Den109G commented on code/modules/map_fluff/celestation.dm:
```diff
@@ -0,0 +1,21 @@
+//Remapped by SAAD (https://github.com/SAADf603) spetial for SS1984
```
Grammar police говорит, что тут должно быть `special`

SAADf603 committed:
fix1

SAADf603 committed:
f1.1

still not ready i just hate this red cross

SAADf603 committed:
mcfix1

SAADf603 committed:
Update celestation.dmm

SAADf603 committed:
f1.1

SAADf603 committed:
f1.2

SAADf603 committed:
add1(areas, auto armsky, robotics teleporter)

SAADf603 committed:
map_changes(armsky, dirt, and some more maps)

SAADf603 committed:
Merge branch 'celestation' of https://github.com/SAADf603/Paradise into celestation

SAADf603 committed:
fix1.3

preopen poddoors, areas, teleporter

SAADf603 committed:
fix1.4

critical bugs like space tiles, pipe missing, non area space and etc

SAADf603 committed:
map_changes2

maintrance, floors, bots(except comand dot),
trying to remember what else, but cant
look mapdif lol

SAADf603 committed:
mapchange fix

Last local tests ver

SAADf603 committed:
fix 1.5

SAADf603 committed:
fix1.6

SAADf603 committed:
fix1.7

SAADf603 committed:
Merge branch 'master220' into celestation

SAADf603 committed:
fast cargo fix

forgot about conveyor

SAADf603 committed:
это такой конец

SAADf603 committed:
Delete celestation.dmm

SAADf603 committed:
Create celestation.dmm

SAADf603 committed:
mapchange

SAADf603 committed:
Update celestation.dmm
